extensions/common/context.cc

```diff
@@ -163,6 +163,7 @@ void populateHTTPRequestInfo(bool outbound, RequestInfo* request_info) {
     request_info->destination_service_host =
         getHeaderMapValue(HeaderMapType::RequestHeaders, kAuthorityHeaderKey)
             ->toString();
+    request_info->destination_service_name = cluster_name;
```

Comment:
not sure if this is a good fallback or not..

Reply:
Does cluster_name mean xDS cluster name? I think parsing host FQDN is preferred over that for consistency (we can still of course extract FQDN from cluster name as an option).

Reply:
Yeah, I was thinking about passthrough and blackhole cluster case, where cluster name does not follow Istio convention and is used for service name. Seems like we need more logic here, I removed this line and added a todo for it. Let's get the bug fix in first.

Reply:
Yes let’s just keep this fix small, just enough to unlock UI.